pages/news/2020-05-12-mixxx-2-3-beta-released.html

```diff
@@ -0,0 +1,84 @@
+title: Mixxx 2.3 beta released
+author: Be.
+date: 2020-05-12 01:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown %}
+We are pround to announce the release of Mixxx 2.3 beta, [download it!](/download/) This release brings hotcue colors & labels, Serato metadata import, Rekordbox metadata import, intro & outro cues, a revamped LateNight skin, multithreaded analysis, deck cloning, and more. A brief overview of the changes is below. In the coming weeks, we will publishing a series of posts on this blog discussing the new features in more depth, so stay tuned!
```

Comment:
```suggestion
We are proud to announce the release of Mixxx 2.3 beta, [download it!](/download/)

When we released 2.2 in 2018, we announced that we we're going to release Mixxx more often. Unfortunately we didn't manage to follow that plan for 2.3, but we're still commited to that goal for future releases.

That said, we think that it was definitely worth the wait:

This release brings hotcue colors & labels, Serato metadata import, Rekordbox metadata import, intro & outro cues, a revamped LateNight skin, multithreaded analysis, deck cloning, and more. A brief overview of the changes is below.

In the coming weeks, we will publishing a series of posts on this blog discussing the new features in more depth, so stay tuned!
```

Reply:
We should also add small paragraph where we ask people to test, report bugs, submit patches, translate and improve the manual.

Reply:
Eh, I don't think we need to add anything about the delay between releases.